modules/hcp-migrate-aws-single-to-multiarch.adoc

```diff
@@ -0,0 +1,168 @@
++
+[source,terminal]
+----
+$ oc patch -n clusters hostedclusters/<hosted_cluster_name> -p '{"spec":{"platform":{"aws":{"multiArch":true}}}}' --type=merge
+----
+
+.. Set the multi-architecture {product-title} release image for the hosted cluster. Ensure that you use the same {product-title} version as the hosted cluster. Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n clusters hostedclusters/<hosted_cluster_name> -p '{"spec":{"release":{"image":"quay.io/openshift-release-dev/ocp-release:<4.x.y>-multi"}}}' --type=merge <1>
+----
+<1> Replace `<4.y.z>` with the supported {product-title} version that you use.
+
+.. Confirm that the multi-architecture image is set in the hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get hostedcluster/<hosted_cluster_name> -o jsonpath='{.spec.release.image}'
+----
+
+. Check that the status of the `HostedControlPlane` resource is `Progressing` by running the following command:
++
+[source,terminal]
+----
+$ oc get hostedcontrolplane -n <hosted_control_plane_namespace> -oyaml
+----
++
+.Example output
+[source,yaml]
+----
+...
+  - lastTransitionTime: "2024-07-28T13:07:18Z"
+    message: HostedCluster is deploying, upgrading, or reconfiguring
+    observedGeneration: 5
+    reason: Progressing
+    status: "True"
+    type: Progressing
+...
+----
+
+. Check that the status of the `HostedCluster` resource is `Progressing` by running the following command:
++
+[source,terminal]
+----
+$ oc get hostedcluster <hosted_cluster_name> -n <hosted_cluster_namespace> -oyaml
+----
+
+.Verification
+
+* Verify that the multi-architecture {product-title} release image is updated in your `HostedControlPlane` resource by running the following command:
++
+[source,terminal]
+----
+$ oc get hostedcontrolplane -n clusters-example -oyaml
+----
++
+.Example output
+[source,yaml]
+----
+version:
+    availableUpdates: null
+    desired:
+      image: quay.io/openshift-release-dev/ocp-release:<4.x.y>-multi <1>
+      url: https://access.redhat.com/errata/RHBA-2024:4855
+      version: 4.16.5
+    history:
+    - completionTime: "2024-07-28T13:10:58Z"
+      image: quay.io/openshift-release-dev/ocp-release:<4.x.y>-multi
+      startedTime: "2024-07-28T13:10:27Z"
+      state: Completed
+      verified: false
+      version: <4.x.y>
+----
+<1> Replace `<4.y.z>` with the supported {product-title} version that you use.
+
+[NOTE]
+====
+The multi-architecture {product-title} release image is updated in your `HostedCluster`, `HostedControlPlane` resources and hosted control plane pods. However, your existing node pools do not transition with the multi-architecture image automatically because, the release image transition is decoupled between hosted cluster, hosted control plane, and node pool. You must create new node pools on your new multi-architecture hosted cluster.
```

Comment:
> the release image transition is decoupled between hosted cluster, hosted control plane, and node pool

I believe the hosted cluster and hosted control plane are coupled with respect to the release image so maybe it should be `the release image transition is decoupled between the hosted cluster/hosted control plane and the node pool`

Reply:
Yes, but I would not mention HostedControlPlane at all as it is not a top level API resource.

`the release image transition is decoupled between the hosted cluster and the node pools`

Reply:
Cool. Updated as suggested!